plugins/dropdown/src/dropdown.tsx

```diff
@@ -0,0 +1,69 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import { Builder } from '@builder.io/sdk'
+import React, { useEffect, useState } from 'react'
+import { InputLabel, MenuItem, FormControl } from '@material-ui/core'
+import Select from '@material-ui/core/Select'
+import { getMassagedProps } from './dropdownPropsExtractor'
+import { safeEvaluate } from './mapperEvaluator'
+import { executeGet } from './selectionsClient'
+
+const comparer = (a: any, b: any) => {
+  if (a.name < b.name) return -1
+  if (a.name > b.name) return 1
+  return 0
+}
+
+export const Dropdown = (props: any) => {
+  const [selections, setSelections] = useState([])
+
+  const onSelectChange = (event: React.ChangeEvent<{ value: unknown }>) => {
+    const selectedValue = event.target.value ?? null
+    props.onChange(selectedValue)
+  }
+
+  const getSelections = async (url: any, mapper: String) => {
+    try {
+      const data = await executeGet(url)
+      const mappedSelections = safeEvaluate(mapper, { data })
+      if (mappedSelections.sort) mappedSelections.sort(comparer)
```

Comment:
maybe the mapper should handle the sorting? in case people want other sorting than alphabetical

e.g. a list of products returned may want to leave the sort the same, since it may be by relevance or such

Reply:
Definitely agree!